marker_map:
  1: [3, 9.000, 4.71238898038469]
  2: [6, 7.400,  3.1416]     
  3: [0, 7.410, 3.1416]       
  4: [6, 4.5, 3.1416]       
  5: [0, 4.500, 0]
  6: [6, 2.720, 3.1416]
  7: [0, 2.720, 0]
  8: [6, 0, 1.5708]
  9: [3, 0, 1.5708]
  10: [0, 0, 1.5708]

axis_markers:
  axis_0_deg: [5, 7]      # Markers facing 0째 (East)
  axis_90_deg: [8, 9, 10, 1]  # Markers facing 90째 (North)
  axis_180_deg: [2, 3, 4, 6]  # Markers facing 180째 (West)
  axis_270_deg: []        # Markers facing 270째 (South)

